src/rsa/verification.rs

```diff
@@ -61,6 +60,86 @@ rsa_pkcs1!(RSA_PKCS1_3072_8192_SHA384, 3072, &super::RSA_PKCS1_SHA384,
            "Verification of signatures using RSA keys of 3072-8192 bits,
             PKCS#1.5 padding, and SHA-384.");
 
+extern {
+    fn GFp_rsa_check_modulus_and_exponent(n: *const bn::BIGNUM,
+                                          e: *const bn::BIGNUM,
+                                          min_bits: c::size_t,
+                                          max_bits: c::size_t) -> c::int;
+}
+
+fn rsa_public_decrypt(out: &mut [u8], public_key_n: &bn::PositiveInteger,
+                      public_key_e: &bn::PositiveInteger, ciphertext: &[u8],
+                      min_bits: usize, max_bits: usize)
```

Comment:
I think we can lose the `rsa_` prefix and in fact I think, especially if we take the approach outlined below, that we should probably just inline this whole function into `ring::rsa::verification::verify_rsa()`.